### DISCUSSION TITLE
Change from SQLite to another database

### DISCUSSION THREAD

**Discussion by devtos** [Chia-Network/chia-blockchain#4071] [Q&A]
Hey, is it possible to change the database from SQLite to MySQL or Postgres? Greetings

**Comment by Akaino** [comment]
Sure. But that would be kinda stupid considering SQLite is so much faster and easier to work with.

**Reply by devtos** [reply]
how can i downvote this non answer?

**Comment by otterslide** [comment]
I'd also like to know if someone can add an option in the config to use Mysql instead.. I could probably do it myself if nobody's willing. I've had sqlite corrupt so many times. Worst case with mysql I had to repair some tables, but sqlite couldn't repair every time I tried. The Chia DB is getting too big to manage as a sqlite for me, I'd rather have it as a mysql and take regular backups. Sqlite backup isn't working for me, the client needs to be shut down.. Mysql can be backed up while running.
